### PR TITLE
Add comment to travis.yml 

### PR DESCRIPTION
**Pull request by jmarlena**

### CODE REVIEW

**Review by aekewu1111 — COMMENTED** [review]
```
The 
```

**Comment by 0640811620** [comment]
2cbbe384b8d0d27af3aacc62ab860be9c7cba273

**Comment by Ed3-m** [comment]
24th of the 10th 2022 to the fourth of the 11th 2022 root 8:30 am Ventnor Isle of Wight

**Review by leem23m — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Travis CI configuration settings
 language: ruby
```
Comment (Shalini9623) [inline .travis.yml:2]: commented at language-shal

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 sudo: false
 addons:
```
Comment (Shalini9623) [inline .travis.yml:4]: line number 4 added test comment

**Review by Mamonaminyar — COMMENTED** [review]
.travis.yml

**Comment by bretthowell714-source** [comment]
Thanks

**Comment by bretthowell714-source** [comment]
Thanks

**Comment by bretthowell714-source** [comment]
Thanks

**Pull request merged**